Sources/NIOFileSystem/BufferedWriter.swift

```diff
@@ -243,3 +243,6 @@ extension WritableFileHandleProtocol {
         }
     }
 }
+
+@available(*, unavailable)
+extension BufferedWriter: Sendable {}
```

Comment:
@glbrntt Same question here

Reply:
Also a yes, this should be conditionally sendable based on the underlying handle